File: 2020/territoriali/tornello/statement/tornello.c
#include <stdio.h>

int solve() {
    int N, i;
    scanf("%d", &N);

    int risposta = 0;  // memorizza qui la risposta

    for (i = 0; i < N; i++) {
        int variazione;
        scanf("%d", &variazione);

        // aggiungi codice...
    }

    return risposta;
}

int main() {
    // se preferisci leggere e scrivere da file
    // ti basta decommentare le seguenti due righe:

    // freopen("input.txt", "r", stdin);
    // freopen("output.txt", "w", stdout);

    int T, t;
    scanf("%d", &T);

    for (t = 1; t <= T; t++) {
        printf("Case #%d: %d\n", t, solve());
    }
}

File: 2020/territoriali/tornello/statement/tornello.pas
function solve(): longint;
var
    N, i: longint;
    variazione: longint;
    risposta: longint;
begin
    readln(N);

    risposta := 0;  (* scrivi qui la risposta *)

    for i:=1 to N do
    begin
        read(variazione);

        (* aggiungi codice... *)
    end;

    solve := risposta;
end;


var
    T, test_index: longint;
begin
    (*
    se preferisci leggere e scrivere da file
    ti basta decommentare le seguenti due righe:
    *)

    (*
    assign(input, 'input.txt'); reset(input);
    assign(output, 'output.txt'); rewrite(output);
    *)

    readln(T);

    for test_index := 1 to T do begin
        readln(); (* ignora la riga vuota *)
        writeln('Case #', test_index, ': ', solve());
    end;
end.
